tutorials/cloud/clustered-redis/deploy-clustered-redis.md

```diff
@@ -0,0 +1,264 @@
+---
+id: deploy-clustered-redis
+summary: Learn how to quickly deploy a multi-node Redis cluster on Ubuntu Server
+categories: cloud
+tags: redis, server, devops, beginner, jaas, juju, aws, azure, compute
+difficulty: 2
+status: published
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+published: 2019-11-10
+author: Tim McNamara <tsm@canonical.com>
+
+---
+
+# Deploy a Redis cluster on Ubuntu Server at any scale 
+
+![](images/welcome.png)
+
+## Welcome
+
+This tutorial walks you through the process of deploying a Redis cluster on Ubuntu Server. The instructions work on all clouds supported by Juju&mdash;even bare metal servers&mdash;but this tutorial includes specific instructions to deploy the cluster to AWS, Google Compute Platform or Azure.
+
+You will be able to configure the exact scale that you require and deploy it into any region supported by each cloud. 
+
+### What you'll learn
+
+* Deploy a Redis cluster
+* Tweak the cluster's settings to add authentication 
+* Test our deployment
+
+### What you'll need
+
+* Juju installed (install it by running `snap install juju --claasic` or by following the instructions on the [Juju documentation](https://jaas.ai/docs/getting-started-with-juju))
+* (Optional) An Ubuntu SSO account (visit [login.ubuntu.com](https://login.ubuntu.com) to create one)
+* (Optional) Credential information for AWS, GCP or Azure (created when you create an account with your preferred cloud provider)
+
+## How this tutorial is structured
+Duration: 0:05
+
+The top of each step provides a command to be executed in the terminal window. Explanatory text then follows for people who want to know more.
+
+To open a terminal window in Ubuntu, run the Terminal activity or holding `Ctrl` + `Alt` + `T` together. 
+
+## (Optional) Log in to JAAS
+Duration: 0:20
+
+```bash
+juju login jaas
+```
+
+### What is "JAAS"?
+
+JAAS is a hosted service provided by Canonical, the company behind Ubuntu. It provides a  "Juju controller" for free. The Juju controller talks to the cloud provider programmatically to provision machines and manage software deployments. Once they're online, the controller then installs the software that we care about: Redis.
+
+
+
+## Create model
+Duration: 0:20
+
+```
+juju add-model redis-cluster-tutorial aws/us-west-1
+```
+
+
+### What's a model and why create one?
+
+A "model" is Juju a term to describe a logical service made up applications. A model might represent a whole web app, including its web server, databases and work queues. 
+
+For our purposes, we'll only be our Redis cluster will make up the entire model. We'll use the name `redis-cluster-tutorial`.
+
```

Comment:
This should probably be moved on top of the command, to provide minimal context.

Reply:
Good point (I went back and forwards on this when I was drafting it, it's good to have an external perspective). Have fixed.